fitbenchmarking/utils/default_options.ini

```diff
@@ -109,6 +109,21 @@ colour_scale: 1.1, #fef0d9
 # default is both
 comparison_mode: both
 
+# table_type selects the types of tables to be produced in FitBenchmarking
+#                 options are 'acc', 'runtime', 'compare'
+#                 'acc' indicates that the resulting table should contain the
+#                       chi_sq values for each of the minimizers
+#                 'runtime' indicates that the resulting table should contain
+#                           the runtime values for each of the minimizers
+#                 'compare' indicates that the resulting table should contain
+#                           both the chi_sq value and runtime values for each
+#                           of the minimizers. The tables produced have the
+#                           chi_sq values on the top line of the cell and the
+#                           runtime on the bottom line of the cell.
```

Comment:
Can you insert a `# default is 'acc', 'runtime', and 'compare'`.
Then we need to copy this to the examples dir, maybe we need to come up with a better way of keeping these up to date with each other.

Reply:
done